app/src/main/java/org/liberty/android/fantastischmemo/utils/HintUtil.java

```diff
@@ -14,20 +16,34 @@
     public HintUtil() {
     }
 
-    public String generateLetterHint(int count, String word) {
+    public static String generateLetterHint(int count, String word, HashSet<Integer> used) {
```

Comment:
why are you overwriting an existing feature?
The feature we had was working well. Why did you not implement a new one instead of overwriting someone else's feature?

Reply:
I'm not overwriting an existing feature, that was the issue that I was assigned to based on what was said during meetings. Same reason as above for why I overwrote the feature since the issue stated that the user could display a random letter from the word and it doesn't make sense for the user to have two different hint options that are so closely related to each other when the other two hint options are different. So I overwrote the existing feature in order to implement the new feature and avoid confusion.